crates/sui-faucet/src/main.rs

```diff
@@ -106,15 +107,14 @@ async fn main() -> Result<(), anyhow::Error> {
                 .into_inner(),
         );
 
-    // TODO (jian):Investigate this issue later.
-    // spawn_monitored_task!(async move {
-    //     info!("Starting task to clear WAL.");
-    //     loop {
-    //         // Every 300 seconds we try to clear the wal coins
-    //         tokio::time::sleep(Duration::from_secs(wal_retry_interval)).await;
-    //         app_state.faucet.retry_wal_coins().await.unwrap();
-    //     }
-    // });
+    spawn_monitored_task!(async move {
+        info!("Starting task to clear WAL.");
+        loop {
+            // Every 300 seconds we try to clear the wal coins
```

Comment:
nit: This comment is assuming a value of `wal_retry_interval`, which is presumably a config -- we should adapt the comment.